units/commit_queue.go

```diff
@@ -66,6 +67,40 @@ func NewCommitQueueJob(env evergreen.Environment, queueID string, id string) amb
 	return job
 }
 
+func (j *commitQueueJob) TryUnstick(cq *commitqueue.CommitQueue) {
+	//unstuck the queue if the patch is done.
+	nextItem := cq.Next()
+	if nextItem == nil {
+		return
+	}
+	patchIdString := nextItem.Issue
+	if !patch.IsValidId(patchIdString) {
+		j.dequeue(cq, nextItem)
+		j.logError(nil, "The Patch id '%s' is not an object id. It was removed from the queue.", nextItem)
```

Comment:
j.logError doesn't log anything if error is nil, so you may need to pass in the first sentence as an Errorf and the second sentence as the message

Reply:
Thanks for spotting that!